python/src/internal.rs

```diff
@@ -163,10 +164,13 @@ impl HudiTable {
         filters: Option<Vec<(String, String, String)>>,
```

Comment:
we should keep python API aligned with rust API. if we introduce a generic filter struct, we would need to bind it over here using pyo3, and then use it in the API

Reply:
Should this be done in another issue? (introducing the generic struct. one of your reviews above mention as well)